### ISSUE TITLE
[models] BN layers call behaviour depends on a 2nd argument

### ISSUE THREAD

**Issue by fg-mindee** [mindee/doctr#47]
According to TF [documentation](https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization#call_arguments_2), BN layers have a second argument in their `call` method which changes its behaviour. This raises the following question:
- how do we pass down this information in a Sequential for instance?
- if it's possible, shouldn't this be a layer attribute that we can switch? (`bn.training = False`) since we won't want to change this behaviour upon each call.
